opennmt/utils/generate_multi_train_configs_a100.py

```diff
@@ -0,0 +1,143 @@
+import argparse
+import copy
+import os
+import random
+
+import yaml
+
+def is_prime(num):
+    if num == 0 or num == 1:
+        return False
+    for x in range(2, num):
+        if num % x == 0:
+            return False
+    else:
+        return True
+
+def update_relative_paths(config: dict):
+    for key, value in config.items():
+        if type(value) == str:
+            if value.startswith('../'):
+                config[key] = '../' + value
+        if type(value) == dict:
+            update_relative_paths(value)
+
+
+def dump_config_to_yaml(dir, config, file_name):
+    if not os.path.exists(dir):
+        os.mkdir(dir)
+    with open(os.path.join(dir, file_name), 'w') as yaml_file:
+        print(f'Creating yaml file {dir}/{file_name}')
+        yaml.dump(config, yaml_file, default_flow_style=False, indent=4, sort_keys=False)
+
+
+def swap_source_target_data(config, dir_name_with_prefix, aug_method=None):
+    new_config = copy.deepcopy(config)
+    for entry in new_config['data'].items():
+        _, entry_value = entry
+        entry_value['path_src'], entry_value['path_tgt'] = entry_value['path_tgt'], entry_value['path_src']
+
+    new_config["test_src"], new_config["test_tgt"] = new_config["test_tgt"], new_config["test_src"]
+
+    src_postfix = config['general']['src_postfix']
+    tgt_postfix = config['general']['tgt_postfix']
+    if aug_method:
+        aug_method_dir_name = os.path.join(dir_name_with_prefix, f'{aug_method}-{tgt_postfix}{src_postfix}')
+    else:
+        aug_method_dir_name = dir_name_with_prefix
+    dump_config_to_yaml(aug_method_dir_name, new_config, 'train_config.yaml')
+
+
+def create_config(config, src_postfix, tgt_postfix, ratio=0, data_size=0, graph_method='base', run=1, seed=1234, threshold=None):
```

Comment:
one small remark, I would suggest to not set a default value to the random seed, if we forgot to set it, the experiments would still run. Let's make it strictly mandatory to set